src/components/Assets/AssetsList.tsx

```diff
@@ -210,13 +213,13 @@ const AssetsList = () => {
     );
 
   let manageAssets = null;
-  if (loading) {
+  if (loading || !data) {
     manageAssets = (
       <div className="col-span-3 w-full py-8 text-center">
         <Loading />
       </div>
     );
-  } else if (assetsExist) {
+  } else if (data?.count) {
```

Comment:
the previous if block already checks if `data` is undefined or falsey.

```suggestion
  } else if (data.count) {
```